kernel/kernel-impl/src/main/java/org/sakaiproject/content/impl/BaseContentService.java

```diff
@@ -7107,8 +7107,12 @@ protected void handleAccessResource(HttpServletRequest req, HttpServletResponse
 						res.addHeader("Content-Length", Long.toString(len));
 					}
 					
-					// SAK-30455: Track event now so the direct link still records a content.read
-					eventTrackingService.post(eventTrackingService.newEvent(EVENT_RESOURCE_READ, resource.getReference(null), false));
+					// SAK-30455: Track event now so the direct link still records a content.read or a content.zip.download
+					if(contentType.equals("application/zip")){
```

Comment:
This should be deleted because it's a different event, let's discuss the right place, should be in the 'zipDownloadItems' method from ResourcesAction.